src/components/App/App.js

```diff
@@ -25,13 +25,17 @@ class App extends Component {
     this.setState({ data });
   }
 
+  handleSelect = (location) => {
+    this.setState({comparedCards: [...this.state.comparedCards, this.district.findByName(location)]});
```

Comment:
Do you think that we should refactor this to make a copy of the object? For example making what you did into a const then passing that in? `const comparedCards = [...this.state.comparedCards, this.district.findByName(location)]
this.setState({ comparedCards })`

Reply:
We wanted a array of objects similar to how we passed the cards to the cardContainer. I set this up the same way. The state is an array of objects. We can discuss this more tomorrow and figure out what we think is the best approach. thanks for reviewing my code.